go/worker/common/p2p/p2p.go

```diff
@@ -111,13 +111,21 @@ func (p *P2P) RegisterHandler(runtimeID common.Namespace, handler Handler) {
 	p.Lock()
 	defer p.Unlock()
 
```

Comment:
Consider updating the method documentation to specify what happens if multiple handlers are registered for a runtime.